src/imageio/storage/email.c

```diff
@@ -202,26 +206,37 @@ void finalize_store(dt_imageio_module_storage_t *self, dt_imageio_module_data_t
     gchar *filename = g_path_get_basename(attachment->file);
     const dt_image_t *img = dt_image_cache_get(darktable.image_cache, attachment->imgid, 'r');
     dt_image_print_exif(img, exif, sizeof(exif));
+
     g_snprintf(body + strlen(body), sizeof(body) - strlen(body), imageBodyFormat, filename, exif);
```

Comment:
Maybe make body a `char*` and use `g_strconcat()` to assemble the string. That way we can't overflow and won't truncate.

Reply:
Done, cleaner anyway.